docker-compose.yml

```diff
@@ -14,3 +14,4 @@ services:
 
   redis:
     image: "redis/redis-stack-server:latest"
+    container_name: "redis-stack-server"
```

Comment:
I don't understand why this is necessary? What is wrong with the default name of `redis`?

Reply:
@mikekelly I have same question.

Reply:
Nothing wrong with it, I could've used redis but in my local testing the image doesn't spin up as redis it's like auto-gpt-redis-1. I could change this to redis but I think specifying a name is useful

Reply:
Just so that there's something consistent people can put in their .env for the host

Reply:
whoop my mistake, no need to specify the name. I at least think the readme update is useful and can modify the readme or close the pr

Reply:
A specific name instead of a generic name helps avoid conflicts and/or confusion when multiple Redis containers may be running on the same host. 

As a best practice, it's better to be specific than generic, especially when you can have OSS Redis, the standard engine; Redis Stack extension with builtin RedisInsight for Dev; and Redis Stack Server, the extension without RedisInsight for Production. 

As convention, using the origin product name plus optional specific identifiers helps any new comers and avoids misunderstandings.

Reply:
@aosan if you decide to go in that direction, you're still better off achieving this through the name of the service. Docker compose provides a network where the containers can address each other via their service name.